test-gen-plugin/src/main/java/com/dylibso/chicory/maven/wast/WasmValue.java

```diff
@@ -165,11 +186,14 @@ public String toExpectedValue() {
                             case I32:
                                 sb.append("Long.parseUnsignedLong(\"" + v + "\")");
                                 break;
+                            case I64:
+                                sb.append("Long.parseUnsignedLong(\"" + v + "\")");
+                                break;
                             case F32:
-                                sb.append(
-                                        "Float.intBitsToFloat(Integer.parseUnsignedInt(\""
-                                                + v
-                                                + "\"))");
+                                sb.append("Integer.parseUnsignedInt(\"" + v + "\")");
+                                break;
```

Comment:
I think we don't need to parse these into Float bits in this case, the raw integer will do